_data/communication.yml

```diff
@@ -9,9 +9,9 @@ websites:
     - name: Basecamp
       url: https://basecamp.com/
       img: basecamp.png
-      tfa: Yes
-      sms: Yes
-      doc: https://basecamp.com/help/guides/you/phone-verification
+      tfa: No
+      twitter: basecamp
+      email_address: email@basecamp.com
```

Comment:
Perhaps it's better to use this email address from https://basecamp.com/about/policies/security/response
```suggestion
      email_address: security@basecamp.com
```

Reply:
@mxxcon "Send urgent or sensitive reports directly to security@basecamp.com. "
I feel like that could possibly be a form of misuse. What do you think?

Reply:
Ya, I guess it might not be the most appropriate email address